microshift_troubleshooting/microshift-troubleshoot-backup-restore.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="microshift-troubleshoot-backup-restore.adoc"]
+= Troubleshoot data backup and restore
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-troubleshoot-data-backup-and-restore
+
+toc::[]
+
+To troubleshoot failed data backups and restorations, check the basics first, such as data paths, storage configuration, and storage capacity.
+
+[id="troubleshoot-backup-restore-microshift-backup-data-failed"]
+== Backup data failed
+Data backups are automatic on OSTree systems. If you are not using an OSTree system and attempted to create a manual backup, the following reasons could cause the backup to fail:
```

Comment:
I noticed `RPM-OSTree` and `OSTree` are used interchangeably - should we settle on one?

Reply:
Yeah, I'm thinking `rpm-ostree` because it is like that in the RHEL for Edge docs.